assets/js/fetch-release.js

```diff
@@ -1,53 +1,99 @@
-function updateDownloads(data) {
-    document.getElementById("wallet-version").textContent  = 'Current Wallet Version: ' + data.name;
+const allPlatforms = ["win64", "win32", "macos_arm", "macos"];
+var platformsMissed = [...allPlatforms];
 
-    var has64BitHotfix = false;
-    var has32BitHotfix = false;
+function show(element){
+    element.style.display = "";
+}
+ 
+function doOtherPlatformsMatch(targetPlatform, name){
+    
+    for (let comparePlatform of allPlatforms){
```

Comment:
`let` has unexpected results in IE. Do we care?

https://caniuse.com/mdn-javascript_statements_let

Reply:
The script doesn't work in IE right now after we dropped support in #332. So not an issue